string task_name
string param_name
---
OpspaceParameter param
bool ok
string errstr
